src/gui/windows/main.zig

```diff
@@ -15,12 +15,16 @@ pub var window = GuiWindow {
 
 const padding: f32 = 8;
 
+fn exitGame(p:usize) void {
+	_ = p;
+	std.process.exit(0);
+}
 pub fn onOpen() void {
 	const list = VerticalList.init(.{padding, 16 + padding}, 300, 16);
 	list.add(Button.initText(.{0, 0}, 128, "Singleplayer", gui.openWindowCallback("save_selection")));
 	list.add(Button.initText(.{0, 0}, 128, "Multiplayer", gui.openWindowCallback("multiplayer")));
 	list.add(Button.initText(.{0, 0}, 128, "Settings", gui.openWindowCallback("settings")));
-	list.add(Button.initText(.{0, 0}, 128, "Exit TODO", .{}));
+	list.add(Button.initText(.{0, 0}, 128, "Exit TODO",  .{.callback = &exitGame}));
```

Comment:
Since you seem to be fixing it, you should probably remove the TODO on the button.